src/theme/scripts/router.js

```diff
@@ -34,8 +34,10 @@ class Router {
   }
 
   async _onLinkClick(event) {
-    if(event.target.tagName !== 'A') return;
-    const link = new URL(event.target.href);
+    // See if what is clicked has an anchor as parent or is a link itself and create an URL of it
+    const link = event.target.closest('a') ? new URL(event.target.closest('a').href) : null;
```

Comment:
You are potentially traversing the DOM tree twice here. You mind adding a temporary variable to hold the result of `closest()`?

Reply:
Done! Good point :)